Packaging/nix/devilutionx.6

```diff
@@ -0,0 +1,134 @@
+.TH DEVILUTIONX 6 "May 2025" "1.5.4" "DevilutionX Community"
+.SH NAME
+devilutionx \- A source port of Diablo and Hellfire
+.SH SYNOPSIS
+devilutionx [OPTION]
+.SH DESCRIPTION
+.I DevilutionX
+is a source port of
+.B Diablo
+and
+.B Hellfire
+that strives to make it simple to run the game while providing engine improvements, bugfixes, and some optional quality of life features.
+.SH OPTIONS
+.TP
+.B \-h, \-\-help
+Print this message and exit.
+.TP
+.B \-\-version
+Print the version and exit.
+.TP
+.B \-\-data\-dir
+Specify the folder of
+.I diabdat.mpq.
+.TP
+.B \-\-save\-dir
+Specify the folder of save files.
+.TP
+.B \-\-config\-dir
+Specify the location of
+.I diablo.ini.
+.TP
+.B \-\-lang
+Specify the language code (e.g.
+.I en
+or
+.I pt_BR
+).
+.TP
+.B \-n
+Skip startup videos.
+.TP
+.B \-f
+Display frames per second.
+.TP
+.B \-\-verbose
+Enable verbose logging.
+.TP
+.B \-\-record \fI<#>\fR
+Record a demo file.
+.TP
+.B \-\-demo \fI<#>\fR
+Play a demo file.
+.TP
+.B \-\-timedemo
+Disable all frame limiting during demo playback.
+.SH GAME SELECTION
+.TP
+.B \-\-spawn
+Force Shareware mode.
+.TP
+.B \-\-diablo
+Force Diablo mode.
+.TP
+.B \-\-hellfire
+Force Hellfire mode.
+.SH HELLFIRE OPTIONS
+Refer to the official documentation for more details on Hellfire options.
```

Comment:
This is probably because of the left over line in --help, there are no extra options for hellfire, we should clean up that message as well :).
```suggestion
```